stride/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "stride",
+    "daemon_name": "strided",
+    "node_home": "$HOME/.strided",
+    "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/Stride-Labs/testnet/infra-test/poolparty/infra/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "ustrd",
+                "fixed_min_gas_price": 0,
+                "low_gas_price": 0,
+                "average_gas_price": 0.025,
```

Comment:
should we set this to 0? this is the default option in keplr

Reply:
yes, set to 0